### PR TITLE
Split Tools with VizTools - up4484 - Part 1

### PR DESCRIPTION
**Pull request by chrieke**
Prepares inheritance changes by splitting tools into VizTools (all plotting, mapping) and Tools (get_blocks etc). Functionality stays the same till part 2.

Part 2 coming later:
The Tools functionality will only be accessible from the `up42` import object. Plotting functionality will only be available for Catalog, Job, Jobtask & Jobcollection.
Code reference Tool section will be removed. The available plotting functionality of each object will be documented specifically in each chapter, the base functions will be documented in the up42 import object section. As suggested by feedback should make the object structure clearer & plotting easier to use.

### CODE REVIEW

**Review by thaisbendixen42 — APPROVED** [review]
cooool!!

File: up42/catalog.py [hunk 1]
```diff
@@ -54,7 +59,7 @@
 }
 
 # pylint: disable=duplicate-code
-class Catalog(Tools):
+class Catalog(VizTools, Tools):
```
Comment (rodrigoalmeida94) [inline up42/catalog.py:62]: Niceee! Gotta love the multiple object inheritance :)

**Review by rodrigoalmeida94 — APPROVED** [review]
Quite a lot of changes, can't pretend to understand the whole thing but the new class hierarchy makes a lot of sense.

**Review by nekhtiari — APPROVED** [review]
Nice

**Pull request merged**